kitty/fonts.c

```diff
@@ -741,7 +741,19 @@ shape(CPUCell *first_cpu_cell, GPUCell *first_gpu_cell, index_type num_cells, hb
     group_state.last_cpu_cell = first_cpu_cell + (num_cells ? num_cells - 1 : 0);
     group_state.last_gpu_cell = first_gpu_cell + (num_cells ? num_cells - 1 : 0);
     load_hb_buffer(first_cpu_cell, first_gpu_cell, num_cells);
-    hb_shape(font, harfbuzz_buffer, NULL, 0);
+
+    if (!render_ligature) {
+        hb_feature_t feature = {
+            .tag = hb_tag_from_string("calt", strlen("calt")),
+            .start = 0,
+            .end = UINT_MAX,
+            .value = 0
+        };
```

Comment:
You can simplify this and use `hb_feature_from_string("-calt")`.

Reply:
Cool, thanks.